src/sys/ioctl/mod.rs

```diff
@@ -97,6 +97,16 @@
 //! Most `ioctl`s have no or little documentation. You'll need to scrounge through
 //! the source to figure out what they do and how they should be used.
 //!
+//! How do I document the generated functions?
```

Comment:
Let's change this to "Documenting generated ioctl functions". I'd also like to request that you add a rationale why this was added (that generated ioctls are public and should have docs and that many crates error on missing documentation).